aws/data_source_aws_dynamodb_table.go

```diff
@@ -167,6 +167,20 @@ func dataSourceAwsDynamoDbTable() *schema.Resource {
 				Type:     schema.TypeInt,
 				Computed: true,
 			},
+			"encrypt_at_rest": {
```

Comment:
😅  Sorry for this, but could you rename all these `encrypt_at_rest` references to `server_side_encryption` so they better match the DynamoDB API `SSE`?

Reply:
Will do